src/app/layout.tsx

```diff
@@ -0,0 +1,30 @@
+"use client";
+
+import { Inter } from "next/font/google";
+// import './globals.css'
```

Comment:
Pls remove this redundant line

Reply:
Thank you for your feedback and I have removed that line